docs/SemanticARC.md

```diff
@@ -128,8 +145,41 @@ In order to simplify this, we will make the following changes:
    * @unowned @safe
    * @forwarding
 
+[EE] We really need a good definition of all these conventions here.
+I strongly believe that the local/global notation would help a lot for a good formal definition of the conventions.
+It's hard for the compiler to reason anything about global RCs. But local RCs are easily statically computable and for me, semantic ARC is all about dealing with local RCs.
```

Comment:
I have thought about this for a while. I think that formally creating the notion of a global or a local reference count at the SIL level is not necessary since one can always discover if a pairing is local or global by noting that a local pairing is one that is not associated with a store_strong. Since we have all of the pairings, we can always understand that.